src/Components/Artwork/GridItem.tsx

```diff
@@ -43,6 +48,19 @@ export const ArtworkGridItem: React.FC<ArtworkGridItemProps> = ({
     ? transform(imageURL, { width, height })
     : { src: "", srcSet: "" }
 
+  const [linkTo, setLinkTo] = useState(artwork.href)
+
+  useEffect(() => {
+    if (!!disableRouterLinking) {
+      setLinkTo(window.location.href)
+    } else {
+      if (linkTo !== artwork.href) {
+        setLinkTo(artwork.href)
+      }
+    }
```

Comment:
To make this lightly less brittle in the future we should guard against `if` conditions in `else` statements (which might not match if the condition changes, leading to a clickable area) and just fall back to an always available URL:

```tsx
} else {
  url = artwork.href
}
```

Reply:
Thanks. We don't need this with `match.location.href`